packages/tests/server/router.test.ts

```diff
@@ -86,4 +87,75 @@ describe('router', () => {
     const result = await caller.foo.bar.foo.bar.foo.bar();
     expect(result).toBe('Hello I am recursive');
   });
+
+  test('combined router object should be accessible', async () => {
```

Comment:
The name of the test implies that the setup would be something like

```js
const bar = t.router({
  foo: t.procedure.query(() => 'Hello I am recursive'),
});

const router = t.router({
  foo: {
    bar,
  },
});
```

In its current shape, it's no different than `multiple nested levels of subrouter should be accessible` one.
Although I believe that even the case I described above is already covered by `multiple nested levels of subrouter with different constructors should be accessible`.